data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg

```diff
@@ -30,7 +30,7 @@
             {PROLOGUE_STAGE1_END}
         [/part]
         [part]
-            story= _ "The raids were mere probes, at first, and the orcs mere rabble. But they grew more numerous, and threatening, and the raiding parties became war-bands and then companies. Then there arose a great warlord among the Bloody Sword tribe, the chieftain called Khazg Black-Tusk; and he raised an army, and besieged Dwarven Doors."
+            story= _ "At first, the orcs disorganized, a mere rabble that easily fell beneath their blade off. Little did the people of Dwarven Doors know that the attacks were merely a probing at their defenses. The raids they grew more numerous and threatening. And soon the raiding parties became war-bands and then companies. At last, one morning, the people of the dwarven doors arose to see a great warlord of the Bloody Sword tribe, Khazg Black-Tusk, and his army around their defenses."
```

Comment:
missing "were" before "disorganized"? Also, "fell beneath their blade off" sounds awkward... off of what? 3rd sentence: if changing "They" to "The raids", get rid of the original "they". 4th sentence: this is a bit too formal of a context to be starting a sentence with "And". 5th sentence: "Dwarven Doors" is a proper noun and requires capitalization.

Reply:
oh wait never mind I see now that this is outdated